sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -34,6 +38,42 @@
 
 namespace SonarAnalyzer.UnitTest.TestFramework
 {
+
+    class ErrorInUnitTest : Exception
+    {
+#pragma warning disable S3963 // "static" fields should be initialized inline
```

Comment:
I'm surprised there isn't also an issue complaining that the exception class is not serializable.